components/RssFeedBundle/README.md

```diff
@@ -79,3 +79,26 @@ system:
                     match:
                         Identifier\ContentType: [article]
 ```
+
+To render meta link tag into your page head :
+```
+{{ render(controller('Novactive\\\EzRssFeedBundle\\Controller\\RssFeedViewController::rssHeadLinkTagsAction'))
+}}
+```
+### Custom SiteListService
+
+As default `Novactive\EzRssFeedBundle\Services\SiteListService` is implimented to fetch for Site Access list within Site Factory
+To do your own implimenation you have to impliment the given Interface :
+`Novactive\EzRssFeedBundle\Services\SiteListInterface` then config your service as following :
+```yaml
+Services:
+  Novactive\EzRssFeedBundle\Services\SiteListServiceInterface: '@your_own.service_alias'
+```
+
```

Comment:
c'est plus utile du coup non ?

Reply:
c'est utile , j'ai implémenté ça sur CMN pour utilisé le site Factory, afin d'avoir les bons Nom sans passé par une trad des sites access

Reply:
https://github.com/Novactive/CMN-cmn-ibexa/pull/412/

Reply:
Ok, du coup faut corriger `implimenation` et cette phrase ` to fetch for Site Access list within Site Factory` qui n'est plus bonne vue que par défaut on utilise plus le service du Site Factory

Reply:
done